### ISSUE TITLE
Rule proposal: No static-only class

### ISSUE THREAD

**Issue by sindresorhus** [sindresorhus/eslint-plugin-unicorn#973]
A class with only static members could just be an object instead.

## Fail

```js
class X {
	static foo = false;
	static bar() {};
}
```

## Pass

```js
const X = {
	foo: false,
	bar() {}
};
```

```js
class X {
	static foo = false;
	static bar() {};

	unicorn() {}
}
```

**Comment by sindresorhus** [comment]
This is now accepted.
